jib-core/src/main/java/com/google/cloud/tools/jib/cache/LayerMetadata.java

```diff
@@ -31,23 +31,23 @@
   static class LayerMetadataEntry {
 
     /** The source file path string, in OS-specific format. */
-    private final String sourceFileString;
+    private final String absoluteSourceFileString;
 
     /** The extraction path string, in OS-specific format. */
-    private final String extractionPathString;
+    private final String absoluteExtractionPathString;
 
-    String getSourceFileString() {
-      return sourceFileString;
+    String getAbsoluteSourceFileString() {
+      return absoluteSourceFileString;
     }
 
-    String getExtractionPathString() {
-      return extractionPathString;
+    String getAbsoluteExtractionPathString() {
+      return absoluteExtractionPathString;
     }
 
     @VisibleForTesting
-    LayerMetadataEntry(String sourceFileString, String extractionPathString) {
-      this.sourceFileString = sourceFileString;
-      this.extractionPathString = extractionPathString;
+    LayerMetadataEntry(String absoluteSourceFileString, String absoluteExtractionPathString) {
```

Comment:
Is there someone verifying these arguments are absolute?

Reply:
Not currently, but the variable names are more of an indication of what to pass in rather than a requirement for being a valid `LayerMetadataEntry`.